Jellyfin.Api/Controllers/UserController.cs

```diff
@@ -498,8 +503,14 @@ public async Task<ActionResult<UserDto>> CreateUserByName([FromBody, Required] C
         [ProducesResponseType(StatusCodes.Status200OK)]
         public async Task<ActionResult<ForgotPasswordResult>> ForgotPassword([FromBody, Required] ForgotPasswordDto forgotPasswordRequest)
         {
+            var ip = HttpContext.GetNormalizedRemoteIp();
```

Comment:
```suggestion
            var ip = context.Connection.RemoteIpAddress ?? IPAddress.Loopback;
```

GetNormalizedRemoteIP converts IPAddress to string - which this is converting back. The above is more optimium and will then use the IPAddress version of the IsInLocalNetwork.

Reply:
These two methods are often used together. I changed the return value of `GetNormalizedRemoteIp` to `IPAddress` and made the `.ToString()`-call where necessary from the calling method